packages/arb-token-bridge-ui/src/components/common/UnsupportedNetworkContent.tsx

```diff
@@ -0,0 +1,41 @@
+import { useAppState } from '../../state'
+import { networksListArray, networkStyleMap } from '../../util/networks'
+import { Button } from './Button'
+
+export const UnsupportedNetworkContent = () => {
+  const {
+    app: { changeNetwork }
+  } = useAppState()
+
+  return (
+    <div className="flex flex-col items-center space-y-8 py-24 px-12">
+      <div className="flex w-full justify-center">
+        <span className="center py-4 text-center text-3xl font-medium text-white">
+          Oops! You’re connected to a wrong network.
+        </span>
+      </div>
+
+      {networksListArray.map(network => (
+        <Button
```

Comment:
visual nit:
the buttons are of different widths
it'd look better if they are all of the same width